docs/manual/notebookashomepage.md

```diff
@@ -69,34 +69,19 @@ That's it! Open your browser and navigate to Apache Zeppelin and see your custom
 If you want to display the list of notes on your custom Apache Zeppelin homepage all
 you need to do is use our %angular support.
 
-Add the following code to a paragraph in you home page and run it... Voila! You have your notes list.
+Add the following code to a paragraph in your Apache Zeppelin note and run it.
```

Comment:
How about adding `%spark` above the code? Since the note might set default interpreter to another one not Spark. I mean
```
%spark

println(
"""%angular
  <div ng-include="'app/home/notebook.html'"></div>
""")
```